### PR TITLE
PS-2253

### PR DESCRIPTION
**Pull request by hilak**
Required email notification for updated entry metadata

### CODE REVIEW

**Comment by hilak** [comment]
@assafb  please review.

File: tests/standAloneClient/metadataObjectChanged.xml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+						<description>Metadata ID</description>
+						<value objectType="KalturaEvalStringField">
+							<code>$scope->getEvent()->getObject()->getId()</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>entry_id</key>
+						<description>Entry ID</description>
+						<value objectType="KalturaEvalStringField">
+							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())) ? $scope->getEvent()->getObject()->getObjectId() : ''</code>
```
Comment (tan-tan-kanarek) [inline tests/standAloneClient/metadataObjectChanged.xml]: No need to check here if is_null, you can always return the getObject()->getObjectId().

File: tests/standAloneClient/metadataObjectChanged.xml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+						<description>Sender name</description>
+						<value objectType="KalturaEvalStringField">
+							<code>kConf::get("partner_notification_name")</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>creator_name</key>
+						<description>Entry creator name</description>
+						<value objectType="KalturaEvalStringField">
+							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())) ? entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getkuser()->getFirstName() . ' ' . entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getkuser()->getLastName() : ''</code>
```
Comment (tan-tan-kanarek) [inline tests/standAloneClient/metadataObjectChanged.xml]: Please use $scope->getObject, no need for the getEvent in the middle, this comment relevant to all the conditions and parameters.

File: tests/standAloneClient/metadataObjectChanged.xml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+				<systemName>EMAIL_ENTRY_METADATA_CHANGED</systemName>
+				<description>Email notification template to be sent when a custom metadata object belonging to an entry changes.</description>
+				<automaticDispatchEnabled>1</automaticDispatchEnabled>
+				<eventType>6</eventType> <!-- EventNotificationEventType::OBJECT_DATA_CHANGED -->
+				<eventObjectType>metadataEventNotifications.Metadata</eventObjectType> <!-- EventNotificationEventObjectType::METADATA -->
+				<eventConditions objectType="array">
+					<item objectType="KalturaFieldMatchCondition">
+						<description>Object type is entry</description>
+						<field objectType="KalturaEvalStringField">
+							<code>$scope->getEvent()->getObject()->getObjectType()</code>
```
Comment (tan-tan-kanarek) [inline tests/standAloneClient/metadataObjectChanged.xml]: Please change $scope->getEvent()->getObject() to $scope->getObject().

**Pull request merged**